### PR TITLE
Fix Baseline bugs (by Tad Glines)

### PR DESCRIPTION
**Pull request by markwgit**
- fix bug in the distro driver code that resulted in false positives for the CheckServiceEnabled code.
- fix gaps in the rpcbind rules for systems running systemd.

@lagalbra 
@NarineM 
@mgladi

### CODE REVIEW

File: installer/conf/omsagent.d/oms_audits.xml [hunk 1]
```diff
@@ -767,9 +767,13 @@
       remediation="Run the command '/usr/local/bin/azsecd remediate -r disable-rpcbind'. This will disable the rpcbind service."
       ruleId="f4a80328-1d67-45ed-b915-274d2e6c699e">
       <check
-        distro="*"
+        distro="Debian|Ubuntu|Oracle|CentOS&lt;7|RedHat&lt;7|SLES=11"
```
Comment (lagalbra) [inline installer/conf/omsagent.d/oms_audits.xml]: How was this list made? Is it supposed to check both distro type and version? If so, we do not only support versions 7 and 11 of CentOS/RedHat/SLES; see full list [here](https://github.com/Microsoft/OMS-Agent-for-Linux/tree/master#supported-linux-operating-systems).

File: installer/conf/omsagent.d/oms_audits.xml [hunk 2]
```diff
@@ -767,9 +767,13 @@
...
         command="CheckServiceDisabled"
         service="rpcbind" />
+      <check
+        distro="CentOS&gt;=7|RedHat&gt;=7|SLES&gt;11"
```
Comment (lagalbra) [inline installer/conf/omsagent.d/oms_audits.xml]: Same distro question here

File: installer/conf/omsagent.d/oms_audits.xml (hunk 1 above)
Comment (markwgit) [inline installer/conf/omsagent.d/oms_audits.xml]: @taglines 
Tad, could you please answer on questions regarding distro and versions

**Comment by markwgit** [comment]
I only tested that new Baseline binaries  and xml are working on different Linux distributions.
I did not changes source ruby files and not FluentD configuration files and event did not changes files names. I only change the content of Baseline scanner files.
These files just copied by build and installation and also does not in use in UTs.

Also, I am planning to test these changes as part of my testing for new agent version.

**Comment by lagalbra** [comment]
@markwgit All changes should be tested before they are merged into master, not only with a new agent version. Related to this, I believe April 21st was the last check-in date for the agent release, so this change will not be included, unless exceptional circumstances are run by @keikhara .

**Comment by lagalbra** [comment]
@markwgit @taglines Are these changes still relevant?

**Pull request closed without merging**